packages/open-next/src/build/compileCache.ts

```diff
@@ -0,0 +1,45 @@
+import path from "node:path";
+
+import * as buildHelper from "./helper.js";
+
+/**
+ * Compiles the cache adapter.
+ *
+ * @param options Build options.
+ * @param format Output format.
+ * @returns The path to the compiled file.
+ */
+export function compileCache(
```

Comment:
This is an added function.

It calls the two existing function for Node and edge if needed.

The edge test has been moved to this function.